plugins/optimization-detective/class-od-url-metric.php

```diff
@@ -406,10 +414,15 @@ public function get_timestamp(): float {
 	/**
 	 * Gets elements.
 	 *
-	 * @return ElementData[] Elements.
+	 * @return OD_Element[] Elements.
 	 */
 	public function get_elements(): array {
```

Comment:
I don't like that `$url_metric->get( 'elements' )` returns an array of arrays (the underlying values), whereas `$url_metric->get_elements()` returns the data wrapped in `OD_Element` objects.

Reply:
Addressed in 5dfcb07e